src/main/drivers/transponder_ir.c

```diff
@@ -84,7 +58,7 @@ static void TRANSPONDER_DMA_IRQHandler(dmaChannelDescriptor_t* descriptor)
     }
 }
 
-void transponderIrHardwareInit(ioTag_t ioTag)
+void transponderIrHardwareInit(ioTag_t ioTag, transponder_t* transponder_instance)
```

Comment:
you can just use `instance` or `transponder` here instead of `transponder_instance`;  you only need a  `transponder_` prefix if there is another instance of something else that would prevent you for using the name `instance` in the same block of code.